app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,40 @@
+<h2>Sign up</h2>
+<div class="register">
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <div class="register_header">
+  <p><%= link_to "Back", root_path %></p>
+  <h3>Register</h3>
+  <div class="actions">
+    <%= f.submit "Next" %>
+  </div>
+</div>
+  <%= render "devise/shared/error_messages", resource: resource %>
+  
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  
+<% end %>
+
+<%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] Its a requirement that we follow the design, kindly do so for the `Login` and `Sign Up` page. Kindly do so

Given design
![image](https://github.com/beteland123/smart_budget/assets/81932478/eefdecc3-8a6d-4b74-9a65-bf23376591c6)

Your design
![image](https://github.com/beteland123/smart_budget/assets/81932478/81e46f0c-2d4a-4f66-abe2-9ef9475c34e7)

![image](https://github.com/beteland123/smart_budget/assets/81932478/cfcb4e4d-4176-4fdc-9b28-cd7e9432a6e7)

- Kindly remove the text to the left of the input fields and instead give the input fields placeholder text
- Also design the input fields to match our given design
- Finally please remove any texts and links after the input fields to match the given design